main.py

```diff
@@ -21,21 +30,25 @@
 
 game_playing = True
 
+input_source = CrowdInput(CrowdInput.standardAddress())
+
 while game_playing:
 
     for event in pygame.event.get():
 
         if event.type == pygame.QUIT:
             game_playing = False
         elif event.type == pygame.KEYDOWN:
-            if event.key == pygame.K_UP:
-                ZUCC.ySpeed = -3
-            elif event.key == pygame.K_DOWN:
+            if event.key == pygame.K_DOWN:
                 ZUCC.ySpeed = 3
             elif event.key == pygame.K_e:
                 ZUCC.evolve()
-        elif event.type == pygame.KEYUP:
-            ZUCC.ySpeed = 0
```

Comment:
Removal of mentions of ZUCC is discouraged. Please increase quantity of ZUCC.

Reply:
:pineapple: